clang/test/CIR/CodeGen/variadics.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o - | FileCheck %s
```

Comment:
Two changes here:
- Please add an extra `RUN` line that also make sure this works for c++20! 
- Instead of sending stuff to stdout and pipe'ing to FileCheck, please use a `%t.cit`. This is handy when investigating testcases breaks, since it makes it easier to open the temporary file in the IDE for investigation. See `binassign.cpp` for an example. Some of the existing tests use stdout, but we should migrate out of it at some point.

Reply:
Applied.

I've had to change to a `cpp` file here.

Reply:
> I had to change to a `cpp` file here.

Alternatively you can also pass to clang `-x c++`

Reply:
Oh, nice. I think I'll use that then.